includes/header.php

```diff
@@ -57,6 +57,7 @@
   <script src="/assets/lib/bootstrap.bundle.min.js"></script>
   <script src="/assets/lib/highlight.min.js"></script>
   <script src="/assets/lib/groovy.min.js"></script>
+  <script src="/assets/lib/mermaid.js"></script>
```

Comment:
move this to page-specific js and check for $includes_mermaid I added to governance.php